### ISSUE TITLE
SPIKE: Remove extra/empty lines in CMS content

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#20187]
## Description

On some pages such as VAMC Stories, there are many empty lines entered in the CMS that come out on the front end as empty paragraphs (`<p>&nbsp;</p>`). This is not great for accessibility and creates large gaps of space, pushing important content further down the page or out of view. It's likely this is caused by editors copying/pasting content from other software into Drupal and not removing the added white space.

Since our design system adds top and bottom spacing to paragraphs by default, we should be able to safely remove any empty paragraphs coming from the CMS to improve accessibility and clean things up.

Given that we are receiving a block of text from the CMS, it may be difficult to traverse it and reliably parse empty `<p>` tags. The goal of this spike is to see if we can make any improvements, or determine that the benefit wouldn't be worth the effort / risk.

This is not an issue with any one content type; it can occur anywhere editors can enter WYSIWYG text and likely will need to be addressed at the root of content-build's data parsing and also in next-build.

Some examples where these spaces occur:

**Extra spaces at the end**
- https://www.va.gov/nebraska-western-iowa-health-care/stories/new-omaha-vamc-inpatient-kitchen-designed-to-improve-workflow-increase-veteran-service/

**Empty `<p>`s within content** - Can we add `aria-hidden="true"` to the `<p>` tags or change these tags to `<br />`?
- https://www.va.gov/boston-health-care/news-releases/va-boston-clinical-researcher-named-as-new-national-academy-of-medicine-member/
- https://www.va.gov/boston-health-care/news-releases/investment-in-well-being-promotion-may-help-reduce-risk-for-ptsd/
- https://www.va.gov/boston-health-care/news-releases/va-boston-housed-more-than-360-veterans-experiencing-homelessness-this-fiscal-year/
- https://www.va.gov/south-texas-health-care/health-services/ (inside "Advice nurse" and "Primary care" accordion)

### User story
> _Describe the audience/user, enhancement or fix, and value / outcome desired._

**AS A**
**I WANT**
**SO THAT**

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** 
- [ ] **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [ ] 
- [ ] Impact analysis is filled out and QA test plan is updated accordingly
- [ ] Does this product have an existing regression test plan?
	- [ ] Yes, link: 
	- [ ] No, regression test plan will be created as part of the work / points in this ticket
